packages/block-library/src/navigation/editor.scss

```diff
@@ -32,7 +32,8 @@
 		padding-right: 0;
 		margin-left: 0;
 		margin-right: 0;
-		margin-bottom: 1em; // Useful for when items wrap.
+		margin-bottom: 0; // Useful for when items wrap.
```

Comment:
"Useful" is a little vague to me coming to this code. Should we consider updating to explain _why_ it's useful?

Reply:
ok, going to research why and how it's _useful_ since I haven't added that comment.